pkg/kp/statusstore/statusstoretest/fake_status_store.go

```diff
@@ -43,24 +50,25 @@ func (s *FakeStatusStore) SetStatus(
 	defer s.mu.Unlock()
 	identifier := StatusIdentifier{t, id, namespace}
 	s.Statuses[identifier] = status
+	s.LastIndex++
 	return nil
 }
 
 func (s *FakeStatusStore) GetStatus(
 	t statusstore.ResourceType,
 	id statusstore.ResourceID,
 	namespace statusstore.Namespace,
-) (statusstore.Status, error) {
+) (statusstore.Status, *api.QueryMeta, error) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
 	identifier := StatusIdentifier{t, id, namespace}
 	status, ok := s.Statuses[identifier]
 	if !ok {
-		return statusstore.Status{}, statusstore.NoStatusError{identifier.String()}
+		return statusstore.Status{}, nil, statusstore.NoStatusError{identifier.String()}
```

Comment:
`(*consulStore).GetStatus()` returns a valid QueryMeta object along with a `NoStatusError` error value. Can you make these behaviors consistent?